company_linkage/airtable_queries/parent_merge.sql

```diff
@@ -0,0 +1,9 @@
+SELECT
+  CSET_id,
+  name,
+  IF(parent_acquisition IS TRUE, TRUE, FALSE) as parent_acquisition,
+  parent_name,
+  parent_id,
+  IF(TRIM(lower(comment)) != "checked", comment, NULL) as comment
```

Comment:
Also, the hotel wifi is awful so I can't feasibly check myself, but if this is meant to be a boolean you might update this

Reply:
Nope, not meant to be a boolean! Just trying to remove some rows that say "checked" and focus on the ones with substantive info in them :) thanks for checking though.